### PR TITLE
Implementation of java.time.YearMonth

### PR DESCRIPTION
**Pull request by alonsodomin**
Class `java.time.YearMonth` submitted for review to the right repo

### CODE REVIEW

**Comment by sjrd** [comment]
This can now be rebased on top of master.

File: .gitignore [hunk 1]
```diff
@@ -15,3 +15,6 @@ project/plugins/project/
 # Scala-IDE specific
 .scala_dependencies
 .worksheet
+
+# IntelliJ specific
```
Comment (nicolasstucki) [inline .gitignore]: Put this on your global gitignore and remove it from here.

**Comment by nicolasstucki** [comment]
I see that you copy and pated the code in your IDE, which trigers a code reformatting. You should consider copying the files directly rather than the content.

**Comment by alonsodomin** [comment]
should be fine now

**Comment by nicolasstucki** [comment]
LGTM

**Pull request merged**